packages/trader/src/AppV2/Components/TradeParameters/trade-parameters.tsx

```diff
@@ -33,11 +34,21 @@ const TradeParameters = observer(({ is_minimized }: TTradeParametersProps) => {
     };
 
     return (
-        <React.Fragment>
+        <div
+            className={clsx(
+                'trade-params__options__wrapper',
+                is_minimized && 'trade-params__options__wrapper--minimized'
+            )}
+        >
+            {is_minimized && isVisible('expiration') && <MultipliersExpirationInfo />}
+            {is_minimized && isVisible('mult_info_display') && <MultipliersDealCancellationInfo />}
+            {is_minimized && isVisible('payout_per_point_info') && <PayoutPerPointInfo />}
+            {is_minimized && isVisible('allow_equals') && <AllowEquals />}
+            {is_minimized && isVisible('payout') && <PayoutInfo />}
```

Comment:
{is_minimized && (
      <>
        {isVisible('expiration') && <MultipliersExpirationInfo />}
        {isVisible('mult_info_display') && <MultipliersDealCancellationInfo />}
        {isVisible('payout_per_point_info') && <PayoutPerPointInfo />}
        {isVisible('allow_equals') && <AllowEquals />}
        {isVisible('payout') && <PayoutInfo />}
      </>
    )}

Reply:
Agree, thank you